tests/installation/test_chooser.py

```diff
@@ -73,7 +73,7 @@ def callback(
 @pytest.fixture()
 def mock_legacy(http: type[httpretty.httpretty]) -> None:
     def callback(
-        request: HTTPrettyRequest, uri: str, headers: dict[str, Any]
+        _: HTTPrettyRequest, uri: str, headers: dict[str, Any]
```

Comment:
revert

Reply:
yep, request is better to describe the param